src/Sulu/Bundle/AdminBundle/Resources/js/views/Tabs/Tabs.js

```diff
@@ -134,8 +135,30 @@ class Tabs<T> extends React.Component<Props<T>> {
                     <TabsComponent onSelect={this.handleSelect} selectedIndex={selectedTabIndex}>
                         {this.sortedTabRoutes.map((tabRoute) => {
                             const tabTitle = tabRoute.options.tabTitle;
+                            const tabBadges = tabRoute.options.tabBadges || [];
+
+                            const badges = (Object.values(tabBadges): any).map((badge: BadgeOptions, index: number) => {
+                                if (typeof badge !== 'object') {
+                                    throw new Error(
+                                        'Badges need to be of type object. This should not happen and is likely a bug.'
```

Comment:
would make this consistent with the toolbar-actions

```suggestion
                                        'The value of a badge entry must be an object, but ' + typeof badge + ' was given!'
```